xblocks_contrib/html/html.py

```diff
@@ -565,3 +564,12 @@ def definition_to_xml(self, resource_fs):
     def non_editable_metadata_fields(self):
         """`use_latex_compiler` should not be editable in the Studio settings editor."""
         return super().non_editable_metadata_fields + [HtmlBlock.xml_attributes, HtmlBlock.use_latex_compiler]
+
+
+class HtmlBlock(HtmlBlockMixin):    # pylint: disable=abstract-method
```

Comment:
Which abstract method, making this class abstract?

Reply:
The `studio_view` method in HtmlBlockMixin just raises NotImplementedError (line 274), so pylint treats it as abstract. to resolve this ignore it.